pokedex/db/load.py

```diff
@@ -434,6 +434,10 @@ def dump(session, tables=[], directory=None, verbose=False, langs=None):
             if langs is None:
                 def include_row(row):
                     return languages[row.local_language_id].official
+            # If the none code is passed, then all the csv files with the local_language_id
+            # column are not updated. In other words they are left blank.
+            elif langs == ['none']:
+                return False
```

Comment:
`none` should be handled in `main.py` too.

Reply:
It can't be handled in main.
If the user enters `pokedex dump -l none` then the parser has to pass `none` as the code to load.py. A code of `none` is not the same as the parser not getting any code. If the parser doesn't get any code, then `pokedex dump` langs defaults to `all`.
The load.py file is the one that figures out what to dump to the csv files depending on the langs code(s). So it checks if no codes are passed (`pokedex dump` OR `pokedex dump -l all` since they are treated the same) and dumps everything to the csvs. If `pokedex dump -l none` is entered then the `return False` from line 440 makes the function only dump the tables that do not contain the `local_language_id` column identifier.
I hope I made this at least somewhat clear, its a bit complicated to explain over plain text.

Reply:
Oh, I missed that detail. You're right that the change i suggested isn't equivalent to the PR's code; however, i don't think that PR's behaviour is correct. The comment in the `dump` function makes it clear that the `langs` argument is only supposed to apply to unofficial translations; official text is always loaded, regardless of language. I think the CLI should preserve that behaviour - `dump -l none` should only disable dumping of translations, not all text. That's consistent with how `load -l` works too.